service/controller/resource/monitoring/remotewriteapiendpointsecret/resource.go

```diff
@@ -57,7 +57,7 @@ func getObjectMeta(ctx context.Context, v interface{}) (metav1.ObjectMeta, error
 
 	return metav1.ObjectMeta{
 		Name:      key.RemoteWriteAgentSecretName,
-		Namespace: key.Namespace(cluster),
+		Namespace: key.ClusterID(cluster),
```

Comment:
So this will not work because the ingress needs the auth part. We need to create 2 secrets here